src/mcp-bundle/doc/index.rst

```diff
@@ -63,6 +63,11 @@ Configuration
                 sse:
                     url: 'http://localhost:8000/sse' # URL to SSE endpoint of MCP server
 
+To add your tools to the MCP server, add the following tag::
+
+    #[AutoconfigureTag('mcp.tool')]
```

Comment:
this is not the right recommendation IMO:

- `AutoconfigureTag` (like all `Autoconfigure*` attributes) is about defining an autoconfiguration rule applying on all child types (its original intent is to be used on interfaces). Recommending to use it to apply a tag on a service directly is a bad idea, as it can have unintended side effects for non-final classes (as it will also impact autoconfigured services defined with child classes)
- if the goal is to automatically tag *autoconfigured* services implementing ToolExecutorInterface with the `mcp.tool` to make it simpler for projects adding a custom tool, the solution is for the MCP bundle to register an autoconfiguration rule for ToolExecutorInterface, so that projects only need to implement the interface and be done with it.